### PR TITLE
Styling phase 2

### PR DESCRIPTION
**Pull request by AmirAsaraf**
1) Extract globals and reused properties
2) Support responsive width

image

image

### CODE REVIEW

File: src/components/react_multi_select.scss [hunk 1]
```diff
@@ -0,0 +1,116 @@
+@import "../resources/globals";
+
+.wrapper {
```
Comment (liorheber) [inline src/components/react_multi_select.scss]: Can we flatten this or is it a limitation of one of our dependencies?

File: src/resources/globals.scss [hunk 2]
```diff
@@ -0,0 +1,35 @@
+// Fonts
+$font-400: 14px;
+$font-300: 12px;
+
+// Spacing
+$space-940: 360px;
```
Comment (liorheber) [inline src/resources/globals.scss]: I know we're using this naming in Kenshoo internally but it doesn't make much sense in my opinion. Sins of the fathers... :(

$font-default-size: 12px;
$font-large-size: 14px;
makes more sense.

The spacings, lets align the param name with the actual value etc.

**Comment by liorheber** [comment]
👍

File: src/resources/globals.scss (hunk 2 above)
Comment (AmirAsaraf) [reply src/resources/globals.scss]: ok

File: src/components/react_multi_select.scss (hunk 1 above)
Comment (AmirAsaraf) [reply src/components/react_multi_select.scss]: We can flatten it. Why not use it as a hierarchy?

Comment (liorheber) [reply src/components/react_multi_select.scss]: I don't mind it but is is meaningless when using css modules with hash, we are importing a specific classes and the hierarchy adds nothing.

Comment (AmirAsaraf) [reply src/components/react_multi_select.scss]: done

File: src/resources/globals.scss (hunk 2 above)
Comment (AmirAsaraf) [reply src/resources/globals.scss]: done

**Comment by liorheber** [comment]
Did you commit anything new?

**Pull request merged**